src/reports/components/formatted-date.tsx

```diff
@@ -19,8 +19,12 @@ export class FormattedDate extends React.Component<FormattedDateProps> {
     }
 
     private formatDateTime(date: Date): string {
-        const moment = Moment.utc(this.props.date);
-        const localMoment = moment.locale(this.props.deps.globalization.languageCode);
-        return localMoment.format('L LTS [UTC]Z');
+        const utcDateTime = DateTime.fromJSDate(date, { zone: 'utc' });
+        const localDateTime = utcDateTime.setLocale(this.props.deps.globalization.languageCode);
```

Comment:
The name `localDateTime` makes me think this has been converted to the local time zone, but that's not the case here; the _locale_ or culture/language has been specified.